config/banners.js

```diff
@@ -39,6 +39,9 @@ export const bannerConfig = {
 			mini: '.wmde-banner-mini-button',
 			form: '.wmde-banner-form-button'
 		},
+		where_does_my_donation_go_link: {
```

Comment:
It'd probably be better to call this `use_of_funds` (as that's what we call the content in the modal) and the sub item isn't a form but a link. Also you can add a selector to check if the modal is open:

```suggestion
		use_of_funds: {
		    link: '.wmde-banner-footer-usage-link',
		    modal_is_open: '.banner_modal.is-visible'
```